installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
+See link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/[Assisted Installer for OpenShift Container Platform] to install {product-title} on a single node. You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
```

Comment:
I don't think this link is visible enough. I recommend adding it after the header:

== Installing {sno} using the Assisted Installer

To install {product-title} on a single node, use the web-based Assisted Installer wizard to guide you through the process and manage the installation.

See the link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/[Assisted Installer for OpenShift Container Platform]  documentation for details and configuration options.

Reply:
Also you should probably use the product-title attribute here.

Reply:
Updated As Suggested.